lib/hanami/server.rb

```diff
@@ -49,11 +49,35 @@ def middleware
       mw = Hash.new { |e, m| e[m] = [] }
       mw["deployment"].concat([::Rack::ContentLength, ::Rack::CommonLogger])
       mw["development"].concat(mw["deployment"] + [::Rack::ShowExceptions, ::Rack::Lint])
+      mw["development"].push(::Shotgun::Static) if enable_shotgun_static?
       mw
     end
 
     private
 
+    # @since x.x.x
+    # @api private
+    def enable_shotgun_static?
+      code_reloading? && shotgun_available?
+    end
+
+    # @since x.x.x
+    # @api private
+    def shotgun_available?
+      begin
+        require 'shotgun'
+        true
+      rescue LoadError
+        false
+      end
```

Comment:
May be is better to remove `begin` and use only `rescue`? What do you think? I.e.:
``` ruby
def shotgun_available?
  require 'shotgun'
  true
rescue LoadError
  false
end
```